packages/atlas-service/src/util.ts

```diff
@@ -2,6 +2,8 @@ import type * as plugin from '@mongodb-js/oidc-plugin';
 import type { PreferencesAccess } from 'compass-preferences-model';
 import { defaultsDeep } from 'lodash';
 import { createHash } from 'crypto';
+import { systemCertsAsync } from 'system-ca';
```

Comment:
This file is shared between renderer / web and main process, so if you're adding this mere, make sure that compass-web still comiles. Preferably we should just have this only in the main.ts (same for crypto, but we have to somewhat polyfill it already, so it's less of an issue)